src/components/PopupWithForm.js

```diff
@@ -6,22 +6,36 @@ export default class PopupWithForm extends Popup {
     this._popupForm = this._popupElement.querySelector(".modal__form");
     this._inputList = this._popupForm.querySelectorAll(".modal__input");
     this._handleFormSubmit = handleFormSubmit;
+    this._submitBtn = this._popupElement.querySelector(".modal__button");
+    this._submitBtnText = this._submitBtn.textContent;
   }
 
   _getInputValues() {
-    const data = {};
+    const cardData = {};
     this._inputList.forEach((input) => {
-      data[input.name] = input.value;
+      cardData[input.name] = input.value;
+      cardData[input.about] = input.value;
     });
-    return data;
+    return cardData;
   }
 
   setEventListeners() {
     super.setEventListeners();
     this._popupForm.addEventListener("submit", (evt) => {
       evt.preventDefault();
       this._handleFormSubmit(this._getInputValues());
-      evt.target.reset();
     });
   }
+
```

Comment:
**Could be improved**

You can make ` setInputValues` in class `PopupWithForm`, which can insert data into inputs:

```js

  setInputValues(data) {
    this._inputList.forEach((input) => {
      // Here you insert the `value` by the `name` of the input
      input.value = data[input.name];
    });
  }



```

So, this way you’ll not have to search the inputs of the profile in `index.js`